src/System.Device.Gpio/System/Device/Pwm/Channels/PwmChannel.Windows.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Device.Pwm
+{
+    public partial class PwmChannel
+    {
+        /// <summary>
+        /// Creates a PWM channel running on Windows 10 IoT.
+        /// </summary>
+        /// <param name="pwmChip">The PWM chip number.</param>
+        /// <param name="pwmChannel">The PWM channel number.</param>
+        /// <param name="frequency">The frequency in hertz.</param>
+        /// <param name="dutyCyclePercentage">The duty cycle percentage represented as a value between 0.0 and 1.0.</param>
+        /// <param name="polarity">The polarity the channel should start in.</param>
+        /// <returns>A PWM channel running on Windows 10 IoT.</returns>
+        public static PwmChannel Create(
+            int pwmChip,
+            int pwmChannel,
+            double frequency,
+            double dutyCyclePercentage = 0.5,
+            PwmPolarity polarity = PwmPolarity.ActiveHigh) =>
```

Comment:
Since we are changing `PwmChannel` to be similar to `I2cDevice`/`SpiDevice`, I wonder if we should create `PwmConnectionSettings`.

```csharp
public sealed class PwmConnectionSettings(
            int pwmChip,
            int pwmChannel,
            double frequency,
            double dutyCyclePercentage = 0.5,
            PwmPolarity polarity = PwmPolarity.ActiveHigh)
```

Therefore, you could create default channels like...

```csharp
PwmConnectionSettings settings = new PwmConnectionSettings(0, 0, 2000);
var pwmChannel = PwmChannel.Create(settings);
```

Makes it a little cleaner when you have more than one PWM channel for a binding, as well.

```csharp
// RGB LED.
var rgbLed = RgbLed(
    new PwmConnectionSettings(0, 0, 2000),   // Red
    new PwmConnectionSettings(0, 1, 2000),   // Green
    new PwmConnectionSettings(0, 2, 2000));  // Blue
rgbLed.SetColor(...);

// Since it creates default channels, they would dispose with binding.
rgbLed.Dispose();

// Could also have overload for PwmChannel objects if needed.  Channels would be disposed outside of binding.
var rgbLed = RgbLed(redPwmChannel, greenPwmChannel, bluePwmChannel);
```